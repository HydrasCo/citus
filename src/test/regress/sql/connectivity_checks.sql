SELECT count(*) FROM citus_check_cluster_node_health() WHERE success IS NOT true;
