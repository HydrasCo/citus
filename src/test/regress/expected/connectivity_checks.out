SELECT count(*) FROM citus_check_cluster_node_health() WHERE success IS NOT true;
 count
---------------------------------------------------------------------
     0
(1 row)

